docs/custom_node_development.md

```diff
@@ -175,9 +175,9 @@ Below you can see the list of fully functional custom nodes embedded in the mode
 | [face blur custom node](https://github.com/openvinotoolkit/model_server/tree/main/src/custom_nodes/face_blur) | `/ovms/lib/custom_nodes/libcustom_node_face_blur.so`|
 
 
-**Example:** 
+**Example:**
 Including built-in [horizontal OCR custom node](https://github.com/openvinotoolkit/model_server/tree/main/src/custom_nodes/horizontal_ocr) in the `config.json` would look like:
-```json
```

Comment:
Why do we remove this?

Reply:
To avoid this issue:

![image](https://github.com/openvinotoolkit/model_server/assets/101244613/2c7de06d-5497-480f-860a-3441953e2aa4)